crates/ruff_linter/src/rules/flake8_bugbear/rules/duplicate_exceptions.rs

```diff
@@ -42,13 +42,19 @@ use crate::registry::Rule;
 #[derive(ViolationMetadata)]
 pub(crate) struct DuplicateTryBlockException {
     name: String,
+    is_star: bool,
 }
 
 impl Violation for DuplicateTryBlockException {
     #[derive_message_formats]
     fn message(&self) -> String {
-        let DuplicateTryBlockException { name } = self;
-        format!("try-except block with duplicate exception `{name}`")
+        let DuplicateTryBlockException { name, is_star } = self;
+        let star = if *is_star {
+            "*".to_string()
+        } else {
+            String::new()
+        };
+        format!("try-except{star} block with duplicate exception `{name}`")
```

Comment:
For obscure reasons, let's do this like:
```rust
if is_star {
  format!("try-except* block with duplicate exception `{name}`")
} else {
  format!("try-except block with duplicate exception `{name}`")
}
```

(These messages get included as-is in the docs, so it's preferable to expand them like that sometimes.)

Reply:
sure thing 👍 
I will adopt this approach with all the rules.